components/link/docs/technology-html.nl.md

```diff
@@ -0,0 +1,30 @@
+<!-- @license CC0-1.0 -->
+
+# HTML
+
+- Gebruik het `<a>` element om links aan te maken.
+- Gebuik niet het `title` attribuut. Sommige browsers gebruiken een tooltip om de tekst uit het `title` attribuut te tonen, die tooltip kan over de tekst heenstaan waardoor het niet leesbaar is.
+- Gebruik nooit een `<button>` of een ander element met een `click` event om zelf een link na te maken.
+- Gebruik `aria-current="page"` wanneer de link verwijst naar de huidige pagina. Voor consistente navigatie ([WCAG eis 3.2.3](https://www.w3.org/TR/WCAG21/#consistent-navigation)) is het beter om links in de navigatie te laten staan dan om geen `<a>` te gebruiken voor de huidige pagina.
+- Vermijd het gebruik van `href="#"`. Dit brengt de focus naar het begin van de pagina en dat is nooit wensbaar.
```

Comment:
Misschien hier nog bij zetten dat `#top` een goed alternatief kan zijn?

Reply:
Ik haal de opmerking wel helemaal weg, het is een incompleet verhaal.